lib/Service/AuthenticationService.php

```diff
@@ -3,9 +3,138 @@
 namespace OCA\OpenConnector\Service;
 
 
+use GuzzleHttp\Client;
+use OAuthException;
+use Symfony\Component\HttpFoundation\Exception\BadRequestException;
 
 class AuthenticationService
```

Comment:
Do we want class docblocks in this repo?

Reply:
There was no docblock yet, we can add it if we want to.

Reply:
This is not a question, we want docblocks